NorwegianRefugeeCouncil/core
# [CORE24-236] feat(participants): Update participant endpoint

Jira ticket: JIRA-CORE24-236


## What have you implemented?

PUT endpoint for updating a participant. It validates the request and calls the PRM engine service.
Added validation also for the POST endpoint that was missing it.

## How to test it?

Create a participant, then call the PUT endpoint with some modifications. Check that the values are updated.

## Extra/next steps
